### PR TITLE
Improve Japanese translation of README

### PR DESCRIPTION
**Pull request by ohno418**
Make some improvements and updates to the Japanese README.

### CODE REVIEW

File: README.ja.md [hunk 1]
```diff
@@ -34,23 +34,23 @@ Swayは沢山のディストリビューションで提供されています。"
...
+_\*コンパイル時の依存_
 
 次のコマンドを実行してください:
 
     meson build
     ninja -C build
     sudo ninja -C build install
 
-logindを使用していないシステムでは、バイナリにsuidを設定する必要があります:
+logindやseatdを使用していないシステムでは、swayバイナリをsuidする必要があります:
```
Comment (kennylevinsen) [inline README.ja.md:49]: Ah. This is technically a correct README fix, but [suid support was removed some time back](https://github.com/swaywm/sway/pull/7044). We should update the English README accordingly.

Comment (ohno418) [reply README.ja.md:49]: Got it. I'll remove this part.

Comment (ohno418) [reply README.ja.md]: Done.

**Comment by ddevault** [comment]
LGTM

**Review by emersion — APPROVED** [review]
Thanks!

**Pull request merged**